### PR TITLE
Update defaultRunnerImage to v2.168.0

### PR DESCRIPTION
**Pull request by chenrui333**
Good to have runner by default running the latest version.

### CODE REVIEW

File: config/default/manager_auth_proxy_patch.yaml [hunk 1]
```diff
@@ -23,3 +23,4 @@ spec:
         args:
         - "--metrics-addr=127.0.0.1:8080"
         - "--enable-leader-election"
+        - "--runner-image=summerwind/actions-runner:v2.168.0"
```
Comment (summerwind) [inline config/default/manager_auth_proxy_patch.yaml:26]: If `defaultRunnerImage` is updated, you don't need to specify this option. Could you delete this line please?

Comment (chenrui333) [reply config/default/manager_auth_proxy_patch.yaml:26]: Yeah, I was struggling a little bit with how to do the setting, maybe we should have it in the documentation somewhere.

With that being said, I think I can remove this line.

**Comment by chenrui333** [comment]
@summerwind should be good to go 😄

File: config/default/manager_auth_proxy_patch.yaml (hunk 1 above)
Comment (summerwind) [reply config/default/manager_auth_proxy_patch.yaml]: I agree with you. I will add usage section to the document, thanks!

**Pull request merged**